experiments/AMIP/modular/cli_options.jl

```diff
@@ -31,10 +31,10 @@ function argparse_settings()
         help = "Method to partition turbulent fluxes. [`PartitionedStateFluxes`, `CombinedStateFluxes`]"
         arg_type = String
         default = "CombinedStateFluxes"
-        "--albedo_from_file"
-        help = "Access land surface albedo information from data file"
-        arg_type = Bool
-        default = true
+        "--albedo_type"
+        help = "Access land surface albedo information from data file. [`function`, `map_static`, `map_temporal`]"
```

Comment:
Perhaps `function` is not the best name here. I can think of some alternatives. What about `spatially_varying` ? I guess `function` does not sound too clear because we're still reading these data from a file (as it used to be the case before), whereas "function" to me sounds more like we would define/prescribe this rather than reading it from a file

Reply:
I think function is the idealized function of latitude here, but we could use `idealized` like Atmos does with Insolation.

Reply:
On balance, it's probably better to be consistent with ClimaLSM.

Reply:
Okay, I'll leave it like this for now